Android/cam/src/main/java/com/applicaster/cam/ui/CamActivity.kt

```diff
@@ -57,12 +57,13 @@ class CamActivity : AppCompatActivity(), IBaseActivity {
     }
 
     override fun close() {
-        ContentAccessManager.contract.onCamFinished()
+        ContentAccessManager.contract.onCamFinished(true)
         finish()
     }
 
     override fun onBackPressed() {
         navigationRouter.callFragmentBackPressed()
+        ContentAccessManager.contract.onCamFinished(false)
```

Comment:
Not sure that to finish the hook by calling this logic inside onBackPressed() method. It fixes issue with "The Player opens and loads endlessly, instead of you being redirected back to the screen you were on"(https://applicaster.atlassian.net/browse/TPS-105) but if I switched to other screen and go back it calls the hook finishing logic with success "false" every time. That means that player will be never opened even if I purchase the item with correct flow.

Reply:
Ineed! Fixed.